sway-fmt-v2/src/utils/attribute.rs

```diff
@@ -21,7 +21,7 @@ impl<T: Parse + Format> Format for Annotated<T> {
     }
 }
 
-trait FormatDecl {
+pub trait FormatDecl {
```

Comment:
```suggestion
pub(crate) trait FormatDecl {
```

We should prefer `pub(crate)` in general unless we are prepared to support downstream use of the API and handle breaking changes to it in semver accordingly.